server/config.js

```diff
@@ -1,4 +1,4 @@
 module.exports = {
-    database: "mongodb://localhost/trainers",
```

Comment:
Remember, this is the database name, so it's descriptive of the entire database, not just the trainers collection. "personal trainer" is what I've been using everywhere so far, but we can refactor if we come up with a better name for the project.